microsoft/testsuites/hpc/infinibandsuit.py

```diff
@@ -120,6 +120,8 @@ def verify_ping_pong(self, environment: Environment, log: Logger) -> None:
         server_node = environment.nodes[0]
         client_node = environment.nodes[1]
 
+        # Ensure RDMA is setup
+        client_node.features[Infiniband]
```

Comment:
I added a new feature called `run_in_parallel`. It speeds up, especially on executing the same operation on different nodes. Please try it out.